packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.ts

```diff
@@ -0,0 +1,49 @@
+import { FASTElement, observable, attr } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "../breadcrumb-item";
+
+/**
+ * A Breadcrumb Custom HTML Element.
+ *
+ * @public
+ */
+export class Breadcrumb extends FASTElement {
+    /**
+     * @internal
+     */
+    @observable
+    public slottedBreadcrumbItems: HTMLElement[];
+    public slottedBreadcrumbItemsChanged() {
+        if (this.$fastController.isConnected) {
+            const lastNode: HTMLElement = this.slottedBreadcrumbItems[
+                this.slottedBreadcrumbItems.length - 1
+            ];
+
+            if (lastNode instanceof BreadcrumbItem) {
+                (lastNode as BreadcrumbItem).separator = false;
+            }
+
+            let childNodeWithHref: HTMLElement | null = null;
+
+            if (lastNode.childElementCount > 0) {
+                lastNode.childNodes.forEach((item: HTMLElement) => {
+                    if (item instanceof HTMLElement && item.hasAttribute("href")) {
+                        childNodeWithHref = item;
+                    }
+                });
+            }
+
+            /**
+             *  If child node with href is found then apply aria-current to child node otherwise apply aria-current to the host element, with an href
```

Comment:
I would do this whole chunk around setting aria-current in a function and call it here, maybe even break the part about finding a child node with an href into its own function as well

Reply:
updated to break up the function